test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -551,7 +551,7 @@ Describe "Invoke-WebRequest tests" -Tags "Feature" {
     It "Validate Invoke-WebRequest error with -Proxy option set - '<name>'" -TestCases $testCase {
         param($proxy_address, $name, $protocol)
 
-        $uri = Get-WebListenerUrl -Test 'Delay' -TestValue '5' -Https:$($protocol -eq 'https')
+        $uri = Get-WebListenerUrl -Test 'Delay' -TestValue '6' -Https:$($protocol -eq 'https')
         $command = "Invoke-WebRequest -Uri '$uri' -TimeoutSec 5 -Proxy '${proxy_address}' -SkipCertificateCheck"
```

Comment:
Since it appears that this test expects a timeout to happen, should we make timeoutsec less?  like 2 seconds?

Reply:
@SteveL-MSFT  I'm good with that. I will modify your previous change as well so they are all on the same settings. Are you thinking the `-TestValue '5'` and the `-TimeoutSec 2`?

Reply:
I think delay of 5 and timeout of 2 makes sense